data/data/aws/vpc/sg-master.tf

```diff
@@ -76,6 +76,46 @@ resource "aws_security_group_rule" "master_ingress_vxlan_from_worker" {
   to_port   = 4789
 }
 
+resource "aws_security_group_rule" "master_ingress_geneve" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.master.id}"
+
+  protocol  = "udp"
+  from_port = 6081
+  to_port   = 6081
+  self      = true
+}
+
+resource "aws_security_group_rule" "master_ingress_geneve_from_worker" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.master.id}"
+  source_security_group_id = "${aws_security_group.worker.id}"
+
+  protocol  = "udp"
+  from_port = 6081
+  to_port   = 6081
+}
+
+resource "aws_security_group_rule" "master_ingress_ovndb" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.master.id}"
+
+  protocol  = "udp"
+  from_port = 6641
+  to_port   = 6642
```

Comment:
should this be 6641?

Reply:
@pecameron IIRC from/to is a port range and on the master we want to allow ingress to both NBDB and SBDB from other masters (but only allow ingress to SBDB by nodes)